cmd/process-agent/subcommands/check/check.go

```diff
@@ -90,6 +90,8 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 			return fxutil.OneShot(runCheckCmd,
 				fx.Supply(cliParams, bundleParams),
 
+				fx.Invoke(command.SetHostMountEnv),
```

Comment:
We should only need to invoke this on the check command, since that is the only command in which gopsutil would be used.